Cython/Compiler/Nodes.py

```diff
@@ -627,7 +629,6 @@ def analyse(self, base_type, env, nonempty=0, visibility=None, in_pxd=False):
         array_type = PyrexTypes.c_array_type(base_type, size)
         return self.base.analyse(array_type, env, nonempty=nonempty, visibility=visibility, in_pxd=in_pxd)
 
```

Comment:
```suggestion
        return self.base.analyse(array_type, env, nonempty=nonempty, visibility=visibility, in_pxd=in_pxd)


```

(would rather leave 2 lines)